Sources/SwiftDocC/Model/Rendering/Tutorial/LineHighlighter.swift

```diff
@@ -134,22 +137,29 @@ public struct LineHighlighter {
             return Highlight(line: offset + 1)
         }
         
-        return Result(file: new, highlights: highlights)
+        let deleteHighlights = diff.removals.compactMap { removal -> Highlight? in
+            guard case .remove(let offset, _, _) = removal else { return nil }
+            // Use 1-based indexing for line numbers.
```

Comment:
Nit: cleanup the comment (it's the same as above). Perhaps extract that little block into a func if it's reused?